ground_truth_slam/plugin.cpp

```diff
@@ -6,10 +6,18 @@
 #include "data_loading.hpp"
 
 #include <chrono>
+#include <filesystem>
+#include <fstream>
 #include <iomanip>
 #include <thread>
 
 using namespace ILLIXR;
+#define ViconRoom1Easy      1403715273262142976
```

Comment:
Maybe add a comment explaining what these are?

Reply:
added, also explained later in the code.